tests/functional/provider_client/test_onboarding_token_generation.py

```diff
@@ -0,0 +1,234 @@
+import logging
+
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    ManageTest,
+    tier4c,
+    skipif_ocp_version,
+    skipif_managed_service,
+    runs_on_provider,
+    skipif_external_mode,
+    yellow_squad,
+    hci_provider_required,
+)
+from ocs_ci.deployment.hosted_cluster import (
+    HostedODF,
+    HostedClients,
+)
+from ocs_ci.deployment.helpers.hypershift_base import (
+    get_random_hosted_cluster_name,
+)
+from ocs_ci.utility.version import get_ocs_version_from_csv
+from ocs_ci.ocs.resources.catalog_source import get_odf_tag_from_redhat_catsrc
+from ocs_ci.utility.utils import (
+    get_latest_release_version,
+)
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.ui.validation_ui import ValidationUI
+from ocs_ci.framework.pytest_customization.marks import ignore_leftovers
+
+log = logging.getLogger(__name__)
+
+
+# @pytest.fixture(scope="class")
+# def setup_ui_class(request):
+#     driver = login_ui()
+
+#     def finalizer():
+#         close_browser()
+
+#     request.addfinalizer(finalizer)
+#     return driver
+
+
+@ignore_leftovers
+@tier4c
+@yellow_squad
+@skipif_ocs_version("<4.15")
+@skipif_ocp_version("<4.15")
+@skipif_external_mode
+@runs_on_provider
+@skipif_managed_service
+class TestOnboardingTokenGeneration(ManageTest):
+    def test_onboarding_token_generation_option_is_available_in_ui(
+        self,
+        setup_ui,
+    ):
+        """
+        Test to verify storage-->storage clients-->Generate client onboarding token
+        option is available in ui
+
+        Steps:
+            1. check onboarding-ticket-key and onboarding-private-key are available
+                under secrets page for openshift-storage ns
+            2. navigate to storage-->storage clients page
+            3. check Generate client onboarding token option is available
+            4. user can generate onboarding token by selecting this option.
+        """
+        secret_ocp_obj = ocp.OCP(
+            kind=constants.SECRET, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
```

Comment:
we need to use `config.ENV_DATA['cluster_namespace']` instead of constant `constants.OPENSHIFT_STORAGE_NAMESPACE`